src/NServiceBus.AcceptanceTests/Routing/When_subscribing_to_a_base_event_with_routes_to_base_and_specific_events_pointing_to_different_publishers.cs

```diff
@@ -0,0 +1,98 @@
+﻿namespace NServiceBus.AcceptanceTests.Routing
+{
+    using System.Threading.Tasks;
+    using AcceptanceTesting;
+    using EndpointTemplates;
+    using Features;
+    using NUnit.Framework;
+
+    public class When_subscribing_to_a_base_event_with_routes_to_base_and_specific_events : NServiceBusAcceptanceTest
```

Comment:
Class name doesn't match file name. I'm guessing this one might be because the name is too long? If so, I'd prefer to see a comment here showing the desired full name.